### DISCUSSION TITLE
Projects based on tags

### DISCUSSION THREAD

**Discussion by Benjamin-Medina** [marcusolsson/obsidian-projects#368] [Help] [answered]
Hello I'm attempting to make a project database based on tags, but am not recieving results from the system. For example, there are 30 notes with the "sapling" tag, according to Obsidians core tags plugin. Projects is unable to detect any of the notes, and I made sure to check my spelling and the inclusion of a "#." Interestingly, I've recieved better results with the dataview function. 

Also, I noticed there were notes missing from the database when the tag does actual work. Projects only shows 186, when there's apparently 350. 

Any help is appreciated. Thank you!

**Comment by marcusolsson** [comment]
Is this related to #364 maybe?

Could you check if the missing notes define the tag in the front matter?

**Reply by Benjamin-Medina** [reply]
Yes, @anvimea brings up a good point, I have those tags in the front matter. Again, Obsidian registers the tags for those in the front matter and in the body of the note.

**Reply by library0012** [reply]
I found an interesting bug within the plugin. Because the obsidian recongnises not only "tag" & "tags", but also "Tag" & "Tags", the uppercase or lowercase of "T" doesn't matter. However, in the plugin,  the uppercase or lowercase of "T" dose matter. And unfortunately I use the "Tags" in my YAML, so the plugin doesn't recognised all my note when I use the tag as the data source. Perhaps I should change all my YAML XDDDD.

**Comment by anvimea** [comment, marked as answer]
I'm having this same issue. Am on the latest version.

What I've noted is that if I use "tag: book" in my frontmatter, it doesn't register. But "tags: book" does register with the plugin.

Unfortunately I have hundreds of book files that use "tag", not "tags" 😅

**Reply by marcusolsson** [reply]
I didn't know Obsidian recognized both `tags` _and_ `tag`. I've added support for `tag` in 1.11.4. Let me know if you're still missing notes, and what format you're using.

**Reply by anvimea** [reply]
Updated and tested it, everything works exactly as expected now. Thanks for the quick fix!

**Reply by Benjamin-Medina** [reply]
Yes, works wonderfully. Thank you so much!

**Comment by pwilkins** [comment]
Simply a comment to other noobs like myself.  In the Project settings for tags, enter the tag as "#your_tag" not "your_tag".  It took me a while to notice the ghost prompt in the textbox.
